src/test/java/com/verisign/storm/metrics/GraphiteMetricsConsumerTest.java

```diff
@@ -184,17 +210,16 @@ public void shouldReadDataAndSendLong(Collection<DataPoint> dataPoints) {
     return new Object[][] { new Object[] { dpList } };
   }
 
-  @Test(dependsOnMethods = { "shouldReadConfigAndContext" }, dataProvider = "generateDataPointsFloat")
+  @Test(dependsOnMethods = { "shouldInitializeGraphiteReporter" }, dataProvider = "generateDataPointsFloat")
```

Comment:
Currently, the tests in `GraphiteMetricsConsumerTest` only ensure that the correct metric names and values are being appended to the `AbstractReporter` buffer. 

The tests for `GraphiteReporter` and `KafkaReporter` verify that the buffered metrics are making it to the Graphite and Kafka backends, respectively.

Maybe some end to end integration tests can be added?